app/views/pages/sandbox.html.slim

```diff
@@ -0,0 +1,46 @@
+- content_for :page_title_prefix do
+  | Sign in to Teaching Vacancies
```

Comment:
This is not relevant for this page, or maybe you want to use the correct content for this